int8 id
int32 timestamp
Pose2D[] poses
